helm/coredns-app/templates/deployment.yaml

```diff
@@ -80,6 +80,15 @@ spec:
           timeoutSeconds: 5
           successThreshold: 1
           failureThreshold: 5
+        readinessProbe:
+          httpGet:
+            path: /ready
+            port: 8181
+            scheme: HTTP
+          initialDelaySeconds: 10
```

Comment:
Strange that delay for readiness is shorter than for liveness probe; curious what's rationale for that?

Typically, liveness probe should be more resilient that readiness, as simple load shedding technique, to avoid complete outage, cascading failure during overload.

Reply:
It was based on the [upstream value](https://github.com/helm/charts/blob/master/stable/coredns/templates/deployment.yaml#L91-L108). This is the initial delay, so it should only affect the initial probe. However, I'm not sure what an optimal setting is here so definitely open to input

Reply:
Whatever was the reason to set liveness probe initial delay to 60, readiness shouldn't be lower than that, so I'd set it to same as for liveness.

To make liveness more resilient and act as load shedding mechanism, liveness failure threshold can be increased e.g. to 7.